lib/squeeze.ml

```diff
@@ -54,6 +54,8 @@ let manage_domain_zero = ref false
 let gib = Int64.(mul 1024L (mul 1024L 1024L))
 let domain_zero_dynamic_min : int64 ref = ref gib  (* 1 GiB minimum for safety *)
 let domain_zero_dynamic_max : int64 option ref = ref None (* static max *)
+let boot_time_host_free_memory_constant_count : int ref = ref 10
```

Comment:
I have been told that memory gets consumed steadily at boot time, i.e. it won't be constant for long if it is not yet stable. Therefore, we don't need to wait for 10 equal samples. I think that 3 should be enough.

Reply:
See the following test result in an AMD host. It keep steady in a period and slightly changes and keep steady again, so we cannot get an accurate value in this case if we reduce the count of the test.
```
2020-02-25_08:23:59 Free: 7737184, Scrub: 0, Dom0Total: 524288, BootFree: 33045888 KiB
2020-02-25_08:24:06 Free: 7737184, Scrub: 0, Dom0Total: 524288, BootFree: 33045888 KiB
2020-02-25_08:24:07 Free: 7737184, Scrub: 0, Dom0Total: 524288, BootFree: 33045888 KiB
2020-02-25_08:24:08 Free: 7737184, Scrub: 0, Dom0Total: 524288, BootFree: 33045888 KiB
2020-02-25_08:24:09 Free: 7737184, Scrub: 0, Dom0Total: 524288, BootFree: 33045888 KiB
2020-02-25_08:24:10 Free: 7737184, Scrub: 0, Dom0Total: 524288, BootFree: 33045888 KiB
2020-02-25_08:24:11 Free: 7737184, Scrub: 0, Dom0Total: 524288, BootFree: 33045888 KiB
2020-02-25_08:24:12 Free: 7737184, Scrub: 0, Dom0Total: 524288, BootFree: 33045888 KiB
2020-02-25_08:24:13 Free: 7737177, Scrub: 0, Dom0Total: 524288, BootFree: 33045860 KiB
2020-02-25_08:24:14 Free: 7737177, Scrub: 0, Dom0Total: 524288, BootFree: 33045860 KiB
2020-02-25_08:24:15 Free: 7737177, Scrub: 0, Dom0Total: 524288, BootFree: 33045860 KiB
2020-02-25_08:24:16 Free: 7737177, Scrub: 0, Dom0Total: 524288, BootFree: 33045860 KiB
2020-02-25_08:24:17 Free: 7737177, Scrub: 0, Dom0Total: 524288, BootFree: 33045860 KiB
2020-02-25_08:24:18 Free: 7737177, Scrub: 0, Dom0Total: 524288, BootFree: 33045860 KiB
2020-02-25_08:24:19 Free: 7737176, Scrub: 0, Dom0Total: 524288, BootFree: 33045856 KiB
2020-02-25_08:24:20 Free: 7737174, Scrub: 0, Dom0Total: 524288, BootFree: 33045848 KiB
2020-02-25_08:24:21 Free: 7737174, Scrub: 0, Dom0Total: 524288, BootFree: 33045848 KiB
2020-02-25_08:24:22 Free: 7737174, Scrub: 0, Dom0Total: 524288, BootFree: 33045848 KiB
2020-02-25_08:24:23 Free: 7737174, Scrub: 0, Dom0Total: 524288, BootFree: 33045848 KiB
2020-02-25_08:24:24 Free: 7737174, Scrub: 0, Dom0Total: 524288, BootFree: 33045848 KiB
2020-02-25_08:24:25 Free: 7737174, Scrub: 0, Dom0Total: 524288, BootFree: 33045848 KiB
2020-02-25_08:24:26 Free: 7737174, Scrub: 0, Dom0Total: 524288, BootFree: 33045848 KiB
2020-02-25_08:24:27 Free: 7737174, Scrub: 0, Dom0Total: 524288, BootFree: 33045848 KiB
2020-02-25_08:24:28 Free: 7737174, Scrub: 0, Dom0Total: 524288, BootFree: 33045848 KiB
```

Reply:
I see, that's a shame... :(